src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const countMale = people.filter(isMan => isMan.sex === 'm');
```

Comment:
Rename the `isMan` variable to `person` for better readability and to reflect that the object represents a person.